dotnet/src/SemanticKernel/Text/SKFunctionTextExtensions.cs

```diff
@@ -5,8 +5,8 @@
 using Microsoft.SemanticKernel.Orchestration;
 
 #pragma warning disable IDE0130
-// ReSharper disable once CheckNamespace - Using NS of ISKFunction
-namespace Microsoft.SemanticKernel.SkillDefinition;
+// ReSharper disable once CheckNamespace - Using main namespace
+namespace Microsoft.SemanticKernel;
```

Comment:
Avoid using `Microsoft.SemanticKernel` when the folder don't represent it.

Not sure why is this in a `Text` folder, the correct place should be close to what it is extending (if in the same package)..

```suggestion
namespace Microsoft.SemanticKernel.Functions;
```